exporter/exporterhelper/queued_retry_v1.go

```diff
@@ -0,0 +1,116 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Would `queued_retry_inmemory.go` be a better name for this file?

Reply:
Good idea, I renamed the file (and so did with `v2`)